lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="certificate-container">
-    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <h2 class="hd-6"><%- gettext('XSeries Program Certificates') %>:</h2>
```

Comment:
That's a good point. It would be great to just rely on the Pattern Library doing the right thing. IMO, though, it is more annoying to have to put a class on every single heading, and those that you forget to put them on end up styling incorrectly. With my Jekyll site, it seemed much easier to have a single rule in my Sass than to try to change everywhere that generated an h2. I wish that the Pattern Library had an optional page level partial that would style all the built-in elements correctly, rather than relying on very careful usage of classes. 

I still hate the ``hd-x`` naming convention though. :-)